### ISSUE TITLE
After fresh install screen ends up with Chrome connection error

### ISSUE THREAD

**Issue by JPMakesStuff** [BillyBlaze/OctoPrint-TouchUI#467]
Fresh install on a Pi 4B, after upgrading things ending up at OctoPrint v1.5.2, OctoPi v0.18.0, TouchUI v0.3.17, Python v3.7.3

To get past the touchscreen (IO header connected, not HDMI connected) only displaying console text and not the browser I followed...

https://github.com/BillyBlaze/OctoPrint-TouchUI/issues/251

and...

https://github.com/BillyBlaze/OctoPrint-TouchUI/issues/365#issuecomment-595192476

...which you say ** DON'T ** change the TOUCHUI_URL, and instead follow...

https://github.com/BillyBlaze/OctoPrint-TouchUI/issues/368

and then doing this...

~/oprint/bin/octoprint config set --bool server.allowFraming true
sudo service octoprint restart
sudo service touchui restart

then I end up with a different error in chrome...

Loading TouchUI...
Startup failed, touch to retry
(if I touch, it goes back to Connecting to TouchUI and stays there forever)

So then I discover this thread: https://github.com/BillyBlaze/OctoPrint-TouchUI/issues/445
Basically says to fudge the TOUCHUI_URL  .... ** AGAIN **

??? what am I to believe is the right fix ???

**Comment by JMHaro** [comment]
I've gone through several installs myself and changing the TOUCHUI_URL as per #445 is what always resolves the issue for me.  Here are my notes for my own setup which has worked every time for me so far.  And I'm on the latest Pi version and Octoprint version currently.

		git clone https://github.com/BillyBlaze/OctoPrint-TouchUI-autostart.git ~/TouchUI-autostart/
		sudo ~/TouchUI-autostart/helpers/install
		Enter username in prompt:  {name}
		cd ~
		cd .octoprint/
		sudo nano config.yaml
		Underneath the autologinAs or autologinLocal lines add:
			    localNetworks:
		            - "127.0.0.1/8"
		            - "::1/128"
		nano ~/TouchUI-autostart/chromium.xinit
			In the CHROME_ARGS= line, Change the $TOUCHUI_URL   to   http://0.0.0.0/

		Disable the TouchUI Screen saver:
			sudo nano /etc/default/touchui
				Change the DISABLE_SCREENSAVER=false  to   DISABLE_SCREENSAVER=true
		sudo reboot now

**Comment by lewisd32** [comment]
I have the same issue, and the fix described by @JMHaro does fix it for me.
Without changing the `$TOUCHUI_URL` in the chrome args, it didn't fix it. But with that, it did.

**Comment by TheMrCaveman** [comment]
MM have the same issue but when issueing the nano ~/TouchUI-autostart/chromium.xinit command there is no CHROME_ARGS line?
what am i doing wrong here

**Comment by mrouillard** [comment]
> MM have the same issue but when issueing the nano ~/TouchUI-autostart/chromium.xinit command there is no CHROME_ARGS line?
> what am i doing wrong here

double check, at the time of writing it is line 93.
